storm-server/src/main/java/org/apache/storm/daemon/nimbus/Nimbus.java

```diff
@@ -4166,4 +4184,23 @@ public void shutdown() {
     public boolean isWaiting() {
         return timer.isTimerWaiting();
     }
+
+    @Override
+    public void processWorkerMetrics(WorkerMetrics metrics) throws org.apache.thrift.TException {
+        if (this.metricsStore == null) {
```

Comment:
It would also be nice to have a metric for the number of times that this method was called, like with the other RPC metrics we have.